package-lock.json

```diff
@@ -3897,6 +3898,9 @@
       "version": "4.0.0",
       "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-4.0.0.tgz",
       "integrity": "sha1-h3Gq4HmbZAdrdmQPygWPnBDjPss=",
+      "dependencies": {
+        "graceful-fs": "^4.1.6"
+      },
       "optionalDependencies": {
         "graceful-fs": "^4.1.6"
       }
```

Comment:
It seems that your installation added those whereas mine didn't. Do you have any idea why it is so?

Reply:
I'm not sure - `npm ls graceful-fs` shows that a lot of packages depend on `graceful-fs@4.2.4`. Maybe the lock generation changed across npm versions? Which version do you use? Mine is 7.4.0

Reply:
I've tried resetting `package-lock.json` to the version currently on master and regenerating it but it produced the same result. 😕

Reply:
Mine is v7.7.6. I've done that once again, and yes, it removed it - I guess it's an OS thing. Also, a newer version of npm removed all `resolved` and `integrity` fields - weird. Let's leave it for now.